data/names.txt

```diff
@@ -3069,80 +3069,16 @@ phrase "bad outcomes"
 
 phrase "pirate"
 	word
-		"Caleuche"
-		"Lady Lovibond"
-		"Flying Dutchman"
-		"Princess Augusta"
-		"Eliza Battle"
-		"Iron Mountain"
-		"Eurydice"
-		"Valencia"
-		"Octavius"
-		"Red Jenny"
-		"Ourang Medan"
-		"Sea Bird"
-		"Resolute"
-		"Mary Celeste"
-		"Resolven"
-		"Zebrina"
-		"Deering"
-		"Bavchimo"
-		"Governor Parr"
-		"Joyita"
-		"Teignmouth Electron"
-		"Ocean Wave"
-		"High Aim"
-		"Jian Seng"
-		"Bel Amica"
-		"Kaz II"
-		"Tai Ching"
-		"Lunatic"
-		"Ryou-Un Maru"
-		"T. T. Zion"
-		"Lyubov Orlova"
-		"Skeleton Key"
-		"Ampoliros"
-		"Demeter"
-		"William Kidd"
-		"Adventure Gallery"
-		"Adventure Prize"
-		"Blackbeard"
-		"Queen Anne's Revenge"
-		"Adventure"
-		"Good Fortune"
-		"Scowerer"
-		"Wyndham Galley"
-		"Speakwell"
-		"Sultana"
-		"Whydah"
-		"Ranger"
-		"Lark"
-		"Flying Dragon"
-		"Golden Fleece"
-		"Trompeuse"
-		"La Nouvelle Trompeuse"
-		"Sea Nymph"
-		"Rising Sun"
-		"Postillion"
 		"Indian Queen"
```

Comment:
It's about this point that my Google-fu failed me and I couldn't tell if these were historical pirate ships or names just made up that sounded "piratey". I've left it as-is.